packages/forma-36-website/src/pages/index.mdx

```diff
@@ -35,7 +35,7 @@ import Resource from './../components/Home/Resource';
   <Resources>
     <Resource
       title="Prototype in Figma with Forma 36 components"
-      description="Open the F36 UI Kit in Figma and create a duplicate. Publish it as a Team library and start prototyping."
+      description="Open the F36 UI Kit in Figma and create a duplicate. You need a Figma account to do this. Publish it as a Team library and start prototyping."
```

Comment:
Maybe we should put this last as everything mentioned requires the account. Let's go with this for now just to quickly do what we can to remove confusion.